javaparser-core-testing/src/test/java/com/github/javaparser/CommentsInserterTest.java

```diff
@@ -63,7 +61,7 @@ void issue624() throws IOException {
 
     @Test
     void issue200EnumConstantsWithCommentsForceVerticalAlignment() {
-        CompilationUnit cu = parse("public enum X {" + EOL +
+        CompilationUnit cu = StaticJavaParser.parse("public enum X {" + EOL +
```

Comment:
There is actually a `TestParser` or so in the project, which is meant to have the best possible configuration for tests.

Reply:
Didn't realise this, cheers! 
Just pushed a change to use it :)